front/chat/src/Components/ChatMessagesList.svelte

```diff
@@ -380,59 +394,72 @@
                                                     <CopyIcon size="13" class="tw-mr-1" />
                                                     {$LL.copy()}
                                                 </span>
+                                                {#if $meStore.isAdmin}
+                                                    <span
+                                                        class="wa-dropdown-item tw-text-pop-red"
+                                                        on:click={() => mucRoom.sendRemoveMessage(message.id)}
+                                                    >
+                                                        <Trash2Icon size="13" class="tw-mr-1" />
+                                                        {$LL.delete()}
+                                                    </span>
+                                                {:else}
```

Comment:
You need to verify that the message comes from the current user to show the "Delete" option, users cannot delete other users message.